### DISCUSSION TITLE
UI/ UX  base set

### DISCUSSION THREAD

**Discussion by peteseyeview** [FundamentalFrequency/.github#44] [UI/UX]
@stonepreston amazing work! 

I'm working from a Mac and have no clue how to compile. I'm currently awaiting the delivery of the PCB and screen. If someone would be so kind as to drop screenshots of the current build, I'd love to take a crack. PNG format, 800x480 if possible.

Thank you!

**Comment by stonepreston** [comment]
its probably best you compile on OS X, as the software is under pretty regular development, so things could change pretty fast. It shouldnt be too difficult to follow the instructions in the LMN-3-DAW repo for building from source on OS X. You will need a C++ compiler (clang on OS X is probably already installed) and you will need to install CMake and git (git is probably installed already as well). You should be able to find videos or further instructions on the topic fairly easily. I dont have a Mac so I cant really provide more than that. But it should be enough to install clang and CMake, and then build using the CMake commands in the LMN-3-DAW readme. 

It looks like CMake provides binaries on their website, and installation instructions [here](https://cmake.org/install/).
